gnucash/gnome/window-reconcile.c

```diff
@@ -313,8 +313,22 @@ recnRecalculateBalance (RecnWindow *recnData)
 }
 
 
+/********************************************************************\
+ * amount_edit_focus_out_cb                                         *
+ *   Callback on focus-out event for statement Ending Balance.      *
+ *   Sets the user_set_value flag true if the amount entered is     *
+ *   different to the calculated Ending Balance as at the default   *
+ *   Statement Date. This prevents the entered Ending Balance       *
+ *   being recalculated if the Statement Date is changed.           *
+ *                                                                  *
+ * Args:   widget         - Ending Balance widget                   *
+ *         event          - event triggering this callback          *
+ *         data           - structure containing info about this    *
+ *                          reconciliation process.                 *
+ * Returns:  False - propagate the event to the widget's parent.    *
+\********************************************************************/
```

Comment:
static function descriptions shouldn't go into the API documentation so please change `/**` to `/* "`to hide it from Doxygen. They're not required, though they're certainly harmless as long as they're not done as Doxygen comments.

Reply:
@jralls, Done. Sorry, live and learn.